tasks/setup.py

```diff
@@ -0,0 +1,66 @@
+import os
+import os.path
+from pathlib import Path
+import getpass
+
+from invoke import task
+from invoke.context import Context
+
+from .config import Config, profile_filename
+from .tool import ask, debug, info, warn
+
+@task
+def setup(ctx: Context):
+    """
+    Setup a local environment interactively
+    """
+    info("🤖 Install Pulumi")
+    os.system("brew install pulumi/tap/pulumi")
+    os.system("pulumi login --local")
+
+    info("🤖 Let's configure your environment for e2e tests! Press ctrl+c to stop me")
+    config = Config()
+    config.configParams = Config.Params()
+    config.configParams.agent = Config.Params.Agent()
+    config.configParams.aws = Config.Params.Aws()
+
+    config.configParams.aws.keyPairName = getpass.getuser()
+    keyPairName = ask(f"🔑 Key pair name - stored in AWS Sandbox, default [{config.configParams.aws.keyPairName}]: ")
+    if len(keyPairName) > 0:
+        config.configParams.aws.keyPairName = keyPairName
+
+    checkKeyPair = ask("Did you create your SSH key on AWS and want me to check it is loaded on your ssh agent when creating manual environments or running e2e tests [Y/N]? Default No: ")
+    if len(checkKeyPair) > 0:
+        config.options = Config.Options()
+        config.options.checkKeyPair = checkKeyPair.lower() == "y" or checkKeyPair.lower() == "yes"
+
+    config.configParams.aws.publicKeyPath = Path.home().joinpath(".ssh", "id_ed25519.pub")
+    publicKeyPath = ask(f"🔑 Path to your public ssh key: default [{config.configParams.aws.publicKeyPath}]")
+    if len(publicKeyPath) > 0:
+        config.configParams.aws.publicKeyPath = publicKeyPath
+    while not os.path.isfile(config.configParams.aws.publicKeyPath):
+        warn(f"{config.configParams.aws.publicKeyPath} is not a falid ssh key")
+        config.configParams.aws.publicKeyPath = ask("Path to your public ssh key: ")
+    
+    config.configParams.agent.apiKey = "00000000000000000000000000000000"
+    apiKey = ask("🐶 Datadog API key - default [00000000000000000000000000000000]: ")
```

Comment:
I don't think we should provide a default API Key as `00000000000000000000000000000000` is not valid. What do you think?

Reply:
This is a chicken and egg issue. With fakeintake there is no need for an api key, but our current VM script requires an API key. Plans is to skip API key requirement when using fakeintake, that is the default, but we need to change the script. So, in the meantime, I suggest configuring a fake API key - we still need an API key as the agent will expect to send one to fakeintake, but fakeintake will validate anything.

Reply:
Indeed, I forgot the fact that fake intake doesn't need a real API KEY.